### DISCUSSION TITLE
Unable to compile AWI-CM3.2

### DISCUSSION THREAD

**Discussion by mmammes** [esm-tools/esm_tools#1190] [ESM-Tools support] [answered]
Hi all, 

I ran into a problem trying to compile `awicm3-v3.2` using `esm_tools`. I used the command:
```
esm_tools comp-awicm3-v3.2
```
It works fine for a bit but then I get a lot of error messages (see attached screenshots). 
I'm working on `DKRZ Levante`. Someone else also tried the same on their account but didn't have any problems.

The path to my `esm_tools` (version `6.35.0`):
```
/work/ab0246/a270254/esm_tools/
```
And the path to `awicm3-v3.2`:
```
/work/ab0246/a270254/model_codes/awicm3-v3.2/
```
I tried deleting and re-downloading the model (via `esm_master get-awicm3-v3.2`) but I still get the same errors.
I would be very grateful for any ideas as to what the problem could be.

Best, Mareike

![error_messages_pt1](https://github.com/esm-tools/esm_tools/assets/149616882/a966ade5-9aaa-43c9-b83f-4722b134441d)
![error_messages_pt2](https://github.com/esm-tools/esm_tools/assets/149616882/49080027-7dac-475f-8556-3a27e2a05b63)

**Comment by JanStreffing** [comment]
Hey @mmammes, I will have a look at the comp folder and get back to you.

**Reply by JanStreffing** [reply]
I cannot reproduce the issue. I can install oasis in awicm3-v3.2 using release 6.35.0. I'll come by to have a closer look.

**Comment by JanStreffing** [comment, marked as answer]
We found that conda installation was providing a netcdf version which was compiled with a different compiler than the one that we point to with the module load in esm_tools. The solution was execute esm_master without loading conda in the .bashrc.

**Reply by mandresm** [reply]
Thanks!
